README.md

```diff
@@ -21,19 +21,6 @@ Run unit tests: `sbt test`
 
 Run integration tests: `sbt it:test`
 
-## To view the RAML
```

Comment:
Instead of deleting the whole section, can we just amend to view OAS with the correct steps. 

See the README in [this PR](https://github.com/hmrc/individual-calculations-api/pull/430/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5) for an example